packages/console/src/onboarding/pages/SignInExperience/options.tsx

```diff
@@ -15,7 +15,6 @@ import Facebook from '../../assets/icons/social-facebook.svg';
 import Kakao from '../../assets/icons/social-kakao.svg';
 import Microsoft from '../../assets/icons/social-microsoft.svg';
 import Oidc from '../../assets/icons/social-oidc.svg';
-import Smal from '../../assets/icons/social-smal.svg';
```

Comment:
nit: Just realized we had a typo in the file name...

Reply:
super mailing attachment loader